modules/installing-ocp-agent-inputs.adoc

```diff
@@ -182,6 +192,10 @@ You must provide the rendezvous IP address when you do not specify at least one
 <3> Optional: Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup. Each host must have a unique hostname supplied by one of these methods.
 <4> Enables provisioning of the {op-system-first} image to a particular device. The installation program examines the devices in the order it discovers them, and compares the discovered values with the hint values. It uses the first discovered device that matches the hint value.
 <5> Optional: Configures the network interface of a host in NMState format.
+ifdef::iscsi-boot[]
```

Comment:
No there's no additional field for iscsi-boot so you can remove this.

However this setting should be set when doing iSCSI booting.
minimalISO: true

In addition, similar to PXE booting, the bootArtifacts can be set
bootArtifactsBaseURL: {{ boot_server_url }}

Reply:
Hey Bob, if you're referring to the `ifdef::iscsi-boot[]` line, that is a conditional statement so that lines 196-197 only appear in this section when it appears on the iSCSI page. Let me know if I'm misunderstanding the feedback.

Also, if you're suggesting that I also add `bootArtifactsBaseURL` to the `agent-config.yaml` in this procedure, could you provide me with an example/fake server URL to use? It seems like the yaml file above uses example values rather than replaceable values

Reply:
OK, sounds good.